models/balancer/ethereum/balancer_v1_ethereum_pools_tokens_weights.sql

```diff
@@ -0,0 +1,132 @@
+{{
+    config(
+        alias='v1_pools_tokens_weights',
```

Comment:
add schema name here, similar to all other models here with version number format

Reply:
and remove v1 in alias, as the schema will handle that